src/Features/VisualBasic/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.vb

```diff
@@ -88,6 +88,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
                 End If
             End If
 
+            If trigger.Kind = CompletionTriggerKind.Insertion AndAlso Not trigger.Character = """"c AndAlso Not trigger.Character = "<"c Then
```

Comment:
I'll add a comment but, in short, the code below this point won't produce useful results when `IsTriggerAfterSpaceOrStartOfWordCharacter` is used to trigger the provider.